buildSrc/src/main/groovy/org/elasticsearch/gradle/precommit/PrecommitTasks.groovy

```diff
@@ -161,10 +161,7 @@ class PrecommitTasks {
 
             targetCompatibility = BuildParams.runtimeJavaVersion.majorVersion
             if (BuildParams.runtimeJavaVersion > JavaVersion.VERSION_13) {
-                project.logger.warn(
-                        "Forbidden APIs does not support Java versions past 13. Will use the signatures from 13 for {}.",
-                        BuildParams.runtimeJavaVersion
-                )
+                // forbidden apis does not yet support java 14 (it will in version 3.0), so we must use java 13 target
```

Comment:
Now that I look at this, why are we basing this on runtime java version? This should simply be hard-coded to be `minimumRuntimeVersion` since that's what we pass to `java -release` so it's effectively impossible to use APIs from a newer JDK anyhow as it'd fail at compile time.

We'll need to of course deal with the fallout of forbiddenapi compatibility when we bump the minimum runtime version but I don't see that happening anytime soon.

Reply:
That's certainly something we should look at, but will likely require more investigation (I recall we tried something like this before and ran into issues, likely with multi release jars). I'm happy to do that as a followup, but this PR still will reduce unnecessary noise that is plaguing all local builds now that we require JDK 14 to run the build.

Reply:
Yeah, the difficulty was exactly multi-release JARs.

Reply:
Cool. I'm good with deferring that for now.